MSAL/src/native_auth/public/MSALNativeAuthPublicClientApplication+Internal.swift

```diff
@@ -119,4 +119,21 @@ extension MSALNativeAuthPublicClientApplication {
         internalChallengeTypes.append(.redirect)
         return internalChallengeTypes
     }
+
+    static func getChallengeTypesFromInternalChallengeTypes(
```

Comment:
I would change the name of this method to something that reflects what it does, and/or add comments. Looking at this one vs. the method above, it's not clear to me what each of them does, and why we need two.

Reply:
Renamed to "convertChallengeTypes", which is more aligned with what it does.